becca-new-app/src/App.js

```diff
@@ -1,18 +1,12 @@
 import React from 'react';
-const todoList = [
-  { id: 1, title: 'Read over assignments'},
-  { id: 2, title: 'Exercise for 1 hour'},
-  { id: 3, title:  'Make dinner'}
-];
+import TodoList from './TodoList';
+import AddTodoForm from './AddTodoForm';
 function App() {
 return (
 <div>
```

Comment:
Please indent your code. Children are always indented from its parent. Children on the same level have the same indentation.

Reply:
```
function App() {
  return (
    <div>
      <h1>Todo List</h1>
      <AddTodoForm/>
      <TodoList/>
    </div>
  );
}
```